website/src/components/gallery/ShowcaseSurveyCard/index.tsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import styles from "./styles.module.css";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import {
+  Card,
+  CardFooter,
+  Button,
+  ToggleButton,
+  Image,
+  Body2,
+  Subtitle1,
+} from "@fluentui/react-components";
+
+function closeCard(parentDiv) {
+  let parent = document.getElementById(parentDiv);
+  parent.style.display = "none";
+  // access localStorage until window is defined
+  if (typeof window !== "undefined") {
+    localStorage.setItem("surveyCardDisplay", parent.style.display);
+  }
```

Comment:
nit.  You might want to create a helper function for all your components to handle loading/saving from localStorage, like

localBrowserSet(key, value)
localBrowserRead(key, value)

Then you can do the `typeof window !== "undefined"` in those functions, instead of repeating that whenever a component uses the browser state.

You can create an issue to address this in the future :)